Makefile

```diff
@@ -101,7 +100,7 @@ install-tools: install-golangci-lint install-setup-envtest install-go-junit-repo
 #    make verify
 verify: install-tools
 	./hack/golangci-lint.sh $(GOTAGS)
-	hack/verify-codegen.sh
+#	hack/verify-codegen.sh
```

Comment:
It's a long story. :smile: 

- Upstream deprecated generate-groups and (among other things) removed execute permission from `generate-internal-groups.sh`
- If I fix that, what it's actually doing is effectively a no-op now that the api packages are gone. Verify verifies by re-generating and then comparing what it generated to what exists, but because nothing else we have generates anything, nothing exists and pkg/generated is empty: 
```
WARNING: generate-groups.sh is deprecated.
WARNING: Please use k8s.io/code-generator/kube_codegen.sh instead.

WARNING: generate-internal-groups.sh is deprecated.
WARNING: Please use k8s.io/code-generator/kube_codegen.sh instead.

Generating clientset for machineconfiguration.openshift.io:v1 at github.com/openshift/machine-config-operator/pkg/generated/clientset
Generating listers for machineconfiguration.openshift.io:v1 at github.com/openshift/machine-config-operator/pkg/generated/listers
Generating informers for machineconfiguration.openshift.io:v1 at github.com/openshift/machine-config-operator/pkg/generated/informers
+ echo 'diffing hack/../pkg against freshly generated codegen'
diffing hack/../pkg against freshly generated codegen
+ ret=0
+ diff -Naupr hack/../_tmp/pkg hack/../pkg
+ cp -R hack/../_tmp/pkg/apihelpers hack/../_tmp/pkg/constants hack/../_tmp/pkg/controller hack/../_tmp/pkg/daemon hack/../_tmp/pkg/operator hack/../_tmp/pkg/server hack/../_tmp/pkg/version hack/../pkg
+ [[ 0 -eq 0 ]]
+ echo 'hack/../pkg up to date.'
hack/../pkg up to date.
+ cleanup
+ rm -rf hack/../_tmp
```
So it's successfully comparing nothing to nothing, which would only be a useful verification step in the event someone accidentally added a types.go somewhere. And even then it would just be like "generated doesn't match exsiting, please updating existing" which is probably not what we want unless we think we're going to start stuffing CRD types back in here again. 

To actually do what I think we'd want (verify that our codegen was properly generated from our API), we'd probably need to duplicate the verification happening in openshift/api and openshift/client-go, but they already verify it? So I don't know if it would be worth duplicating that here?

Reply:
Ah, thanks for the context! I think that makes sense, I'm fine with removing it then